factory.json

```
@@ -32,6 +32,10 @@
   "infra": {
     "not_these_templates": ["PULL_REQUEST_TEMPLATE.md"]
   },
+  "storage": {
+    ".gitignore": ["/node_modules/"],
```

So I think that for now we want to ignore the lockfile as well given that we will generate locally and compare the output on git.

---

Yeah, let's do that.

```suggestion
    ".gitignore": ["/node_modules/", "/package-lock.json"],
```